### PR TITLE
Update redirected URLs (part 32)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Extending #14055

## Summary

Updated redirected URLs.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/svg/svg_animation_with_smil/index.md [hunk 1]
```diff
@@ -118,4 +118,4 @@ Same example as before with a curved path and following the direction of the pat
 
 - [SVG](/en-US/docs/Web/SVG)
 - [SVG Animation Specification](https://www.w3.org/TR/SVG/animate.html)
-- [SMIL Specification](https://www.w3.org/TR/REC-smil)
+- [SMIL Specification](https://www.w3.org/TR/REC-smil//)
```
Comment (MrBrain295) [inline files/en-us/web/svg/svg_animation_with_smil/index.md:121]:
```suggestion
- [SMIL Specification](https://www.w3.org/TR/REC-smil/)
```

Comment (MrBrain295) [reply files/en-us/web/svg/svg_animation_with_smil/index.md:121]: Or does it need two slashes?

Comment (OnkarRuikar) [reply files/en-us/web/svg/svg_animation_with_smil/index.md]: good catch thanks!

File: files/en-us/web/xpath/functions/floor/index.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ The nearest integer less than or equal to `number`.
 
 ### Defined
 
-[XPath 1.0 4.4](https://www.w3.org/TR/xpath#function-floor)
+[XPath 1.0 4.4](https://www.w3.org/TR/xpath/#function-floor)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/floor/index.md:29]:
```suggestion
[XPath 1.0 4.4](https://www.w3.org/TR/1999/REC-xpath-19991116/#function-floor)
```

File: files/en-us/web/xpath/functions/false/index.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ This function is useful part of a comparison:
 
 ### Defined
 
-[XPath 1.0 4.3](https://www.w3.org/TR/xpath#function-false)
+[XPath 1.0 4.3](https://www.w3.org/TR/xpath/#function-false)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/false/index.md:34]:
```suggestion
[XPath 1.0 4.3](https://www.w3.org/TR/1999/REC-xpath-19991116/#function-false)
```

File: files/en-us/web/xpath/functions/count/index.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ An integer representing the number of nodes in a node-set.
 
 ### Defined
 
-[XPath 1.0 4.1](https://www.w3.org/TR/xpath#function-count)
+[XPath 1.0 4.1](https://www.w3.org/TR/xpath/#function-count)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/count/index.md:29]:
```suggestion
[XPath 1.0 4.1](https://www.w3.org/TR/1999/REC-xpath-19991116/#function-count)
```

File: files/en-us/web/xpath/functions/contains/index.md [hunk 5]
```diff
@@ -28,7 +28,7 @@ contains(haystack, needle)
 
 ### Defined
 
-[XPath 1.0 4.2](https://www.w3.org/TR/xpath#function-contains)
+[XPath 1.0 4.2](https://www.w3.org/TR/xpath/#function-contains)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/contains/index.md:31]:
```suggestion
[XPath 1.0 4.2](https://www.w3.org/TR/1999/REC-xpath-19991116/#function-contains)
```

File: files/en-us/web/xpath/functions/concat/index.md [hunk 6]
```diff
@@ -26,7 +26,7 @@ A single string that is the concatenation of all the strings passed to the funct
 
 ### Defined
 
-[XPath 1.0 4.2](https://www.w3.org/TR/xpath#function-concat)
+[XPath 1.0 4.2](https://www.w3.org/TR/xpath/#function-concat)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/concat/index.md:29]:
```suggestion
[XPath 1.0 4.2](https://www.w3.org/TR/1999/REC-xpath-19991116/#function-concat)
```

File: files/en-us/web/xpath/functions/ceiling/index.md [hunk 7]
```diff
@@ -32,7 +32,7 @@ for example:
 
 ### Defined
 
-[XPath 1.0 4.4](https://www.w3.org/TR/xpath#function-ceiling)
+[XPath 1.0 4.4](https://www.w3.org/TR/xpath/#function-ceiling)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/ceiling/index.md:35]:
```suggestion
[XPath 1.0 4.4](https://www.w3.org/TR/1999/REC-xpath-19991116/#function-ceiling)
```

File: files/en-us/web/xpath/functions/boolean/index.md [hunk 8]
```diff
@@ -33,7 +33,7 @@ Boolean `true` or `false` after evaluating `expression`.
 
 ## Defined
 
-[XPath 1.0 4.3](https://www.w3.org/TR/xpath#function-boolean)
+[XPath 1.0 4.3](https://www.w3.org/TR/xpath/#function-boolean)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/boolean/index.md:36]:
```suggestion
[XPath 1.0 4.3](https://www.w3.org/TR/1999/REC-xpath-19991116/#function-boolean)
```

File: files/en-us/web/xpath/functions/unparsed-entity-url/index.md [hunk 9]
```diff
@@ -25,7 +25,7 @@ The URI of the unparsed entity retrieved from the DTD, if it exists. Otherwise a
 
 ### Defined
 
-[XSLT 1.0 12.4](https://www.w3.org/TR/xslt#function-unparsed-entity-uri)
+[XSLT 1.0 12.4](https://www.w3.org/TR/xslt/#function-unparsed-entity-uri)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/unparsed-entity-url/index.md:28]:
```suggestion
[XSLT 1.0 12.4](https://www.w3.org/TR/1999/REC-xslt-19991116/#function-unparsed-entity-uri)
```

File: files/en-us/web/xpath/functions/system-property/index.md [hunk 10]
```diff
@@ -32,7 +32,7 @@ An object representing the given system-property.
 
 ### Defined
 
-[XSLT 1.0 12.4](https://www.w3.org/TR/xslt#function-system-property)
+[XSLT 1.0 12.4](https://www.w3.org/TR/xslt/#function-system-property)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/system-property/index.md:35]:
```suggestion
[XSLT 1.0 12.4](https://www.w3.org/TR/1999/REC-xslt-19991116/#function-system-property)
```

File: files/en-us/web/xpath/functions/key/index.md [hunk 11]
```diff
@@ -34,7 +34,7 @@ This function is an XSLT-specific addition to XPath. It is not a part of the cor
 
 ### Defined
 
-[XSLT 1.0 12.2](https://www.w3.org/TR/xslt#function-key)
+[XSLT 1.0 12.2](https://www.w3.org/TR/xslt/#function-key)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/key/index.md:37]:
```suggestion
[XSLT 1.0 12.2](https://www.w3.org/TR/1999/REC-xslt-19991116/#function-key)
```

File: files/en-us/web/xpath/functions/generate-id/index.md [hunk 12]
```diff
@@ -36,7 +36,7 @@ This function is an XSLT-specific addition to XPath. It is not a part of the cor
 
 ### Defined
 
-[XSLT 1.0 12.4](https://www.w3.org/TR/xslt#function-generate-id)
+[XSLT 1.0 12.4](https://www.w3.org/TR/xslt/#function-generate-id)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/generate-id/index.md:39]:
```suggestion
[XSLT 1.0 12.4](https://www.w3.org/TR/1999/REC-xslt-19991116/#function-generate-id)
```

File: files/en-us/web/xpath/functions/function-available/index.md [hunk 13]
```diff
@@ -26,7 +26,7 @@ Boolean `true` or `false`.
 
 ### Defined
 
-[XPath 1.0 15](https://www.w3.org/TR/xslt#function-function-available)
+[XPath 1.0 15](https://www.w3.org/TR/xslt/#function-function-available)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/function-available/index.md:29]:
```suggestion
[XSLT 1.0 15](https://www.w3.org/TR/1999/REC-xslt-19991116/#function-function-available)
```

File: files/en-us/web/xpath/functions/format-number/index.md [hunk 14]
```diff
@@ -35,7 +35,7 @@ This function is an XSLT-specific addition to XPath. It is not a part of the cor
 
 ### Defined
 
-[XSLT 1.0 12.3](https://www.w3.org/TR/xslt#function-format-number)
+[XSLT 1.0 12.3](https://www.w3.org/TR/xslt/#function-format-number)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/format-number/index.md:38]:
```suggestion
[XSLT 1.0 12.3](https://www.w3.org/TR/1999/REC-xslt-19991116/#function-format-number)
```

File: files/en-us/web/xpath/functions/element-available/index.md [hunk 15]
```diff
@@ -26,7 +26,7 @@ Returns true if and only if the expanded-name is the name of an instruction. If
 
 ### Defined
 
-[XSLT 1.0 15](https://www.w3.org/TR/xslt#function-element-available)
+[XSLT 1.0 15](https://www.w3.org/TR/xslt/#function-element-available)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/element-available/index.md:29]:
```suggestion
[XSLT 1.0 15](https://www.w3.org/TR/1999/REC-xslt-19991116/#function-element-available)
```

File: files/en-us/web/xpath/functions/document/index.md [hunk 16]
```diff
@@ -46,7 +46,7 @@ This function is an XSLT-specific addition to XPath. It is not a part of the cor
 
 ### Defined
 
-[XSLT 1.0 12.1](https://www.w3.org/TR/xslt#function-document)
+[XSLT 1.0 12.1](https://www.w3.org/TR/xslt/#function-document)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/document/index.md:49]:
```suggestion
[XSLT 1.0 12.1](https://www.w3.org/TR/1999/REC-xslt-19991116/#function-document)
```

File: files/en-us/web/xpath/functions/current/index.md [hunk 17]
```diff
@@ -65,7 +65,7 @@ the `.` returns the `bar` node, which may be different from the current node.
 
 ### Defined
 
-[XSLT 1.0 12.4](https://www.w3.org/TR/xslt#function-current)
+[XSLT 1.0 12.4](https://www.w3.org/TR/xslt/#function-current)
```
Comment (teoli2003) [inline files/en-us/web/xpath/functions/current/index.md:68]:
```suggestion
[XSLT 1.0 12.4](https://www.w3.org/TR/1999/REC-xslt-19991116/#function-current)
```

File: files/en-us/web/xslt/using_the_mozilla_javascript_interface_to_xsl_transformations/index.md [hunk 18]
```diff
@@ -73,7 +73,7 @@ Transforming XHTML should work as expected though.
 
 ### Setting parameters
 
-You can control [parameters for the stylesheet](https://www.w3.org/TR/xslt#variables) using the {{domxref("XSLTProcessor.setParameter()")}}, {{domxref("XSLTProcessor.getParameter()")}}, and {{domxref("XSLTProcessor.removeParameter()")}} methods. These all take a namespace URI and a local name as the first two parameters, with {{domxref("XSLTProcessor.setParameter()")}} taking a third - the value of the parameter to be set. See [The XSLT/JavaScript Interface in Gecko](/en-US/docs/Web/XSLT/XSLT_JS_interface_in_Gecko/Setting_Parameters) for an example.
+You can control [parameters for the stylesheet](https://www.w3.org/TR/xslt///#variables) using the {{domxref("XSLTProcessor.setParameter()")}}, {{domxref("XSLTProcessor.getParameter()")}}, and {{domxref("XSLTProcessor.removeParameter()")}} methods. These all take a namespace URI and a local name as the first two parameters, with {{domxref("XSLTProcessor.setParameter()")}} taking a third - the value of the parameter to be set. See [The XSLT/JavaScript Interface in Gecko](/en-US/docs/Web/XSLT/XSLT_JS_interface_in_Gecko/Setting_Parameters) for an example.
```
Comment (teoli2003) [inline files/en-us/web/xslt/using_the_mozilla_javascript_interface_to_xsl_transformations/index.md:76]:
```suggestion
You can control [parameters for the stylesheet](https://www.w3.org/TR/1999/REC-xslt-19991116/#variables) using the {{domxref("XSLTProcessor.setParameter()")}}, {{domxref("XSLTProcessor.getParameter()")}}, and {{domxref("XSLTProcessor.removeParameter()")}} methods. These all take a namespace URI and a local name as the first two parameters, with {{domxref("XSLTProcessor.setParameter()")}} taking a third - the value of the parameter to be set. See [The XSLT/JavaScript Interface in Gecko](/en-US/docs/Web/XSLT/XSLT_JS_interface_in_Gecko/Setting_Parameters) for an example.
```

File: files/en-us/web/xslt/using_the_mozilla_javascript_interface_to_xsl_transformations/index.md [hunk 19]
```diff
@@ -46,7 +46,7 @@ You can use the {{domxref("XSLTProcessor.transformToDocument()")}} or {{domxref(
 var newDocument = processor.transformToDocument(domToBeTransformed);
 ```
 
-The resultant object depends on the [output method](https://www.w3.org/TR/xslt#output) of the stylesheet:
+The resultant object depends on the [output method](https://www.w3.org/TR/xslt///#output) of the stylesheet:
```
Comment (teoli2003) [inline files/en-us/web/xslt/using_the_mozilla_javascript_interface_to_xsl_transformations/index.md:49]:
```suggestion
The resultant object depends on the [output method](https://www.w3.org/TR/1999/REC-xslt-19991116/#output) of the stylesheet:
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
XPath and XSLT links were pointing to an indirection page and no more to the spec itself. So all anchors were broken and the link was useless.

I fixed this by skipping this indirection and linking to the actual spec: all anchors now work.

**Review by teoli2003 — APPROVED** [review]
Thanks!

**Comment by OnkarRuikar** [comment]
@teoli2003 Thanks a lot!

**Pull request merged**